src/stores/PersonStore.ts

```diff
@@ -338,5 +344,24 @@ export const usePersonStore: StoreDefinition<'personStore', PersonState, PersonG
         this.loading = false;
       }
     },
+
+    async changePersonInfoById(personId: string, personalnummer: string): Promise<void> {
+      this.loading = true;
+      try {
+        const personByPersonalnummerBodyParams: PersonByPersonalnummerBodyParams = {
+          personalnummer: personalnummer,
+          revision: this.currentPerson?.person.revision ?? '',
+          lastModified: this.currentPerson?.person.lastModified ?? '',
```

Comment:
Since I'm not really intimately familiar with the store-pattern, I feel like getting the current person here might cause problems in some cases. I'd much rather take `revision` and `lastModified` as parameters.

But if this is the normal design pattern, I'm fine with it

Reply:
We can indeed add another computed variable to the vue, extract the revision and last modified from the state of the store and push them as paramter to the calling method but that's just more code I figured. The state of the store is reactive and there can only be 1 currentPerson at the time so it should be okay. Moreover, we don't need revision and lastModified in the Vue.